src/modules/outliers/index.js

```diff
@@ -0,0 +1,82 @@
+import { getZScoreHelper, STANDARD_Z_SCORE } from './zScore'
+import { getModZScoreHelper, MODIFIED_Z_SCORE } from './modZScore'
+import { getIQRHelper, IQR } from './iqr'
+import { normalizerMap, XY_RATIO } from './normalization'
+import { getXYStats } from './xyStats'
+
+export const THRESHOLD_FACTOR = 1.5
+
+export const defaultConfig = {
+    thresholdFactor: THRESHOLD_FACTOR,
+    normalizationMethod: XY_RATIO,
+    outlierMethod: IQR,
+    extremePercentage: 1,
+}
+
+const getDataWithNormalization = (data, normalizationMethod) => {
+    const normalizer = normalizerMap[normalizationMethod]
+
+    if (typeof normalizer !== 'function') {
+        throw `Normalization method ${normalizationMethod} not supported`
```

Comment:
Should this be translated?
```suggestion
        throw i18n.t('Normalization method {{normalizationMethod}} not supported', { normalizationMethod })
```

Reply:
Not sure if it should be there. It's not meant for the user.